packages/remark-parse/lib/parser.js

```diff
@@ -5,17 +5,14 @@
  * @module remark:parse
  * @fileoverview Markdown parser.
  */
+import xtend from 'xtend';
```

Comment:
`lodash.merge` might make a good replacement. I wouldn't switch entirely to `Object.assign` since that requires polyfilling in many environments